apollo-router/src/services/router/service.rs

```diff
@@ -877,15 +887,21 @@ impl RouterService {
                 .and_then(|s| s.as_str())
                 .unwrap_or_default()
                 .to_string();
+            let severity = error.extensions.get("severity").and_then(|s| s.as_str());
             let path = match &error.path {
                 None => "".into(),
                 Some(path) => path.to_string(),
             };
+            // TBD(tim): do we want to also transmit warnings to the user-facing "apollo.router.graphql_error" instrument?
+            // Can see arguments for both sides...
```

Comment:
Right now the PR *does* count it, right?

I think counting it in graphql_error is good, but we should call it out in the changelog so users can understand why their errors are increasing.

Reply:
Sounds good to me.  I'll remove the TBD.